### PR TITLE
Tcjr/new/posts by date order

### PR DESCRIPTION
**Pull request by tomcarmichael**

### CODE REVIEW

File: api/models/post.js
Comment (pablisch) [inline api/models/post.js:1]: Good to see 👍🏼

File: frontend/src/components/feed/Feed.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ const Feed = ({ navigate }) => {
   const [posts, setPosts] = useState([]);
   const [token, setToken] = useState(window.localStorage.getItem("token"));
 
+  function orderByDate (posts) {
+    return posts.sort((a,b) => new Date(a.createdAt) - new Date(b.createdAt)).reverse()
```
Comment (pablisch) [inline frontend/src/components/feed/Feed.js:9]: Wow... I really need to think about this!
What is createdAt? A string I guess?

**Review by pablisch — APPROVED** [review]
Nice work 👍🏼

File: frontend/src/components/feed/Feed.js (hunk 1 above)
Comment (tomcarmichael) [reply frontend/src/components/feed/Feed.js:9]: yes! it comes back from the database as a string but converts perfectly to a Date object in Javasript - it is formatted correctly already

**Pull request merged**